backend/events/models.py

```diff
@@ -26,11 +26,14 @@
 class Event(CreationDeletionMixin):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255)
-    tagline = models.CharField(max_length=255, blank=True)
+    tagline = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
In django CharField and TextField are never saved as NULL values, therefore there is no need for null=True.